### DISCUSSION TITLE
a

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#13823] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [x] My issue happened while using mods.

### What happened?

Content/Items/Shipwrecks/StructurePrefabsWrecked.xml : line 590
![Screenshot_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/16212b89-081f-4eb3-a729-307d7522f72e)

### Reproduction steps

Go check

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by LePiromano** [comment]
literally unplayable

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/e52f81304c6bcf6f53fa373d3400dc4ed29c161c
